src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -48,10 +95,19 @@ export function DetailsPaneComponent({
   graphStyle,
   frameHeight
 }: DetailsPaneComponentProps): JSX.Element {
+  const moreStep = 1000
```

Comment:
have we tested this with any data that has over 1000 properties?

Reply:
yes, see query in comment in this PR.